src/libmongoc/examples/cmake/vcpkg/.gitignore

```
@@ -0,0 +1 @@
+_build/
```

```suggestion
_build/

```
Missing EOF newline.

The root `.gitignore` file already contains `_build/`; does that not apply here?

---

Yeah, this file can be removed.